javascript/commons/Collapse.js

```diff
@@ -132,22 +131,14 @@ liquipedia.collapse = {
 					toggleGroup.classList.remove( 'toggle-state-hide' );
 					toggleGroup.classList.add( 'toggle-state-show' );
 					button.innerHTML = this.makeIcon( true ) + ' ' + showAllText;
-					toggleGroup.querySelectorAll( '.collapsible, .general-collapsible' ).forEach( ( collapsible ) => {
-						collapsible.classList.add( 'collapsed' );
-					} );
-					toggleGroup.querySelectorAll( '.brkts-matchlist-collapsible' ).forEach( ( collapsible ) => {
-						collapsible.classList.add( 'brkts-matchlist-collapsed' );
```

Comment:
This was using a different collapsed class, i'm not sure if that matters? I can't find `brkts-matchlist-collapsed` in the repo to begin with.

Reply:
You could probably check the skin or one of the extensions (or ask rathoz), perhaps that specific class is either redundant or even unused by now.

Reply:
Then we should remove it here as well, was enough time for pages to be purged IMO